Source/OxyPlot/Rendering/OxyRect.cs

```diff
@@ -16,12 +16,12 @@ namespace OxyPlot
     /// <summary>
     /// Describes the width, height, and point origin of a rectangle.
     /// </summary>
-    public struct OxyRect : IFormattable, IEquatable<OxyRect>
+    public readonly struct OxyRect : IFormattable, IEquatable<OxyRect>
```

Comment:
This may have perf implications, but should be fine.

Reply:
If anything it should improve performance as far as I know (because it potentially avoids defensive copies). But I doubt this will be noticeable.

Reply:
I've seen significant performance degradation with `readonly` structs in the past, but that was years ago (netcore 2.x era I think) and as you say it should hopefully go the other way around.